### PR TITLE
[update] [refactor] - layout changes

### PR DESCRIPTION
**Pull request by AnandBFRS01918**

### CODE REVIEW

File: src/app/components/global-search/global-search.component.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+::ng-deep .modal-backdrop {
+  --bs-backdrop-bg: transparent;
```
Comment (vipin-shiprocket) [inline src/app/components/global-search/global-search.component.scss:2]: you don't need this. Just check the documentation of Angular material dialog

File: src/app/app.component.ts [hunk 2]
```diff
@@ -5,4 +5,11 @@ import { Component } from '@angular/core';
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
-export class AppComponent {}
+export class AppComponent {
+  constructor() {
+    const preload = document.createElement('link');
+    preload.rel = 'preconnect';
+    preload.href = window.location.origin;
```
Comment (vipin-shiprocket) [inline src/app/app.component.ts]: why we are doing this?

File: src/app/pages/layout/header/header.component.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<mat-toolbar class="o2-header" style="z-index: 100">
```
Comment (vipin-shiprocket) [inline src/app/pages/layout/header/header.component.html]: z-index is necessary?

File: src/app/pages/layout/layout.component.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+@Component({
+  selector: 'app-layout',
+  templateUrl: './layout.component.html',
+  styleUrls: ['./layout.component.scss'],
+})
+export class LayoutComponent {
+  sidenaveOpen = false;
+  iconRegistry = inject(MatIconRegistry);
+  domSanitizer = inject(DomSanitizer);
+  layoutService = inject(LayoutService);
```
Comment (vipin-shiprocket) [inline src/app/pages/layout/layout.component.ts:16]: you can directly use this instead of sidenaveOpen

File: src/app/app.component.ts (hunk 2 above)
Comment (AnandBFRS01918) [reply src/app/app.component.ts]: preconnect for optimized image. Need to look into this. Commenting for now to revisit later

File: src/app/components/global-search/global-search.component.scss (hunk 1 above)
Comment (AnandBFRS01918) [reply src/app/components/global-search/global-search.component.scss:2]: Temporary addition. Subject to change on receiving actual design

File: src/app/pages/layout/header/header.component.html (hunk 3 above)
Comment (AnandBFRS01918) [reply src/app/pages/layout/header/header.component.html]: removed

File: src/app/pages/layout/layout.component.ts (hunk 4 above)
Comment (AnandBFRS01918) [reply src/app/pages/layout/layout.component.ts:16]: done

File: src/app/app.component.ts (hunk 2 above)
Comment (AnandBFRS01918) [reply src/app/app.component.ts]: Commented the code. Need to revisit to look for another strategy.

**Pull request merged**